content/blog/summary-from-the-9th-annual-chapel-implementers-and-users-workshop-chiuw-2022.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Summary from the 9th Annual Chapel Implementers and Users Workshop (CHIUW 2022)
+date: 2022-07-15T19:31:06.645Z
+priority: 2
+author: Michelle Strout and Engin Kayraklioglu
+authorimage: /img/combined-author-photos.jpg
+thumbnailimage: /img/chapel-logo.jpg
+tags:
+  - Chapel Programming Language
+  - GPU Programming
+  - High Performance Computing
+  - Open Source
+  - Arkouda (Data Analytics in Python powered by Chapel)
+---
+**Introduction**
+
+Programming today becomes complicated by the many kinds of parallelism that exist in everything from phones to laptops to supercomputers.  The open-source Chapel parallel programming language makes parallel programming easier and more productive, while still enabling high performance that takes advantage of the wide variety of parallelism available today.
+
+In this post, Michelle Strout, general workshop chair, and Engin Kayraklioglu, program committee chair, summarize the highlights of the recent 9th Annual Chapel Implementers and Users Workshop (CHIUW 2022). Read on to hear about some exciting applications.  Read on to hear about some exciting applications that are using Chapel productively, the coding day that happened the day before the workshop, updates on the project, and feedback the team received during and after the workshop.
+
+**Applications Written in Chapel**
+
+Programmers generally enjoy trying out new programming languages, but like to see example use cases before they lean on it.  CHIUW provides them with an opportunity to hear about and ask questions about many different use cases.  The Chapel programming language is being used productively in a range of application domains: data science, aeronautical simulations, cosmology simulations, and quantum diagonalization to name a few. CHIUW featured the following talks on different Chapel applications:
+
+* [Large-Scale and User-Friendly Exact Diagonalization in Chapel](<https://youtu.be/vBxPTzIRRr0>)
+* [Recent Developments in the CHApel Multi-Physics Simulation Software] (<https://youtu.be/uTE_RZkODOk>)
```

Comment:
Please make sure to remove the space between the the closed bracket and the first parentheses.
[Recent Developments in the CHApel Multi-Physics Simulation Software] (<https://youtu.be/uTE_RZkODOk>)